src/config.js

```diff
@@ -13,6 +13,8 @@ const config = {
   postgres_port: process.env.POSTGRES_PORT || 5432,
   base_local_url_graphql: `${configGraphQL.local_url}:${configGraphQL.local_port}`,
   base_auth_url_graphql: `${configGraphQL.auth_internal_url}:${configGraphQL.auth_internal_port}`,
+  postgres_user_database: process.env.USER_DATABASE_NAME || 'dojot_dash_users',
```

Comment:
change to BACKSTAGE_DB_NAME and add BACKSTAGE_DB_USER,  BACKSTAGE_DB_PWD, BACKSTAGE_DB_HOST